Description: migrate to SDL2
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-04-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/main/sdl2/input.hpp
+++ b/src/main/sdl2/input.hpp
@@ -10,7 +10,7 @@
 
 #pragma once
 
-#include <SDL.h>
+#include <SDL2/SDL.h>
 
 class Input
 {
--- a/src/main/main.cpp
+++ b/src/main/main.cpp
@@ -9,7 +9,7 @@
 #include <iostream>
 
 // SDL Library
-#include <SDL.h>
+#include <SDL2/SDL.h>
 
 // SDL Specific Code
 #include "sdl2/timer.hpp"
--- a/src/main/sdl2/audio.hpp
+++ b/src/main/sdl2/audio.hpp
@@ -17,7 +17,7 @@
 #pragma once
 
 #include "globals.hpp"
-#include <SDL.h>
+#include <SDL2/SDL.h>
 
 #ifdef COMPILE_SOUND_CODE
 
--- a/src/main/sdl2/renderbase.hpp
+++ b/src/main/sdl2/renderbase.hpp
@@ -3,7 +3,7 @@
 #include "../stdint.hpp"
 #include "../globals.hpp"
 
-#include <SDL.h>
+#include <SDL2/SDL.h>
 
 // Abstract Rendering Class
 class RenderBase
@@ -76,4 +76,4 @@
     int shadow_multi;
 
     bool sdl_screen_size();
-};
\ No newline at end of file
+};
--- a/src/main/sdl2/timer.hpp
+++ b/src/main/sdl2/timer.hpp
@@ -1,6 +1,6 @@
 #pragma once
 
-#include <SDL.h>
+#include <SDL2/SDL.h>
 
 class Timer
 {
@@ -31,4 +31,4 @@
     //Checks the status of the timer
     bool is_started();
     bool is_paused();
-};
\ No newline at end of file
+};
--- a/src/main/sdl2/rendergl.hpp
+++ b/src/main/sdl2/rendergl.hpp
@@ -13,7 +13,7 @@
 #pragma once
 
 #include "renderbase.hpp"
-#include <SDL_opengl.h>
+#include <SDL2/SDL_opengl.h>
 
 class Render : public RenderBase
 {
--- a/src/main/sdl2/audio.cpp
+++ b/src/main/sdl2/audio.cpp
@@ -16,8 +16,8 @@
 
 #include <iostream>
 #include <cstring>
-#include <SDL.h>
 
+#include <SDL2/SDL.h>
 #include "sdl2/audio.hpp"
 #include "frontend/config.hpp" // fps
 #include "engine/audio/osoundint.hpp"
